ROMFS/px4fmu_common/init.d-posix/10017_quadrotor_x

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+#
+# @name Quadrotor SITL model for JSBSim
+#
+# @type Quadrotor Wide
+#
+# @maintainer Jaeyoung Lim <jaeyoung@auterion.com>
+#
+
+sh /etc/init.d/rc.mc_defaults
+
+set MIXER quad_w
```

Comment:
Does this need to exist? Can we use 10016?

Reply:
@dagar I added the airframe for two reasons - the target needs to be the same name as the make target / and the vehicle tuning needed to be different

Reply:
Ok, I understand the current situation with referencing the model for what to run, but for tuning I see lots of identical empty airframes. To me it seems like we need a configuration mechanism that describes the simulator, model, and px4 (airframe, maybe optional parameters) rather than carrying every single variation like this. Maybe I just invented roslaunch...

Reply:
Yeap, totally agree and I agree that it is slowly getting out of hand actually on this note - if we move away from running sitl with make targets it might be a bit easier